src/plugins/data/common/query/persistable_state.test.ts

```diff
@@ -8,33 +8,52 @@
 
 import { extract, inject } from './persistable_state';
 import { Filter } from '@kbn/es-query';
```

Comment:
This is a new file, but a diff is broken because old file was moved:

src/plugins/data/common/query/persistable_state.ts -> src/plugins/data/common/query/filters/persistable_state.ts